standalone/hooks/hook-pysiaf.py

```diff
@@ -0,0 +1,4 @@
+from PyInstaller.utils.hooks import collect_data_files, copy_metadata
+
+datas = collect_data_files('pysiaf')
```

Comment:
What does this mean? pysiaf is not optional anymore?

Reply:
There's no way we can make it optional in the standalone executable, given that all the dependencies are packaged together by pyinstaller. We either include it or we don't - I can't remember if not including it led to an error in the standalone build, but I figure we should have the full functionality available in the executable version?

Reply:
It's still optional if you install from pypi or from source.